### DISCUSSION TITLE
Updates issue?

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#2038] [General]
Went to see if there are any updates from ghelper, it says there is a bios update available. But on Asus site there is no update for bios.
![Screenshot 2024-02-07 124455](https://github.com/seerge/g-helper/assets/149981353/5cc65fff-897b-41b3-bdcf-ca51333c18a1)

**Comment by seerge** [comment]
@crazywhiteboy911 Asus API clearly offers 319

https://rog.asus.com/support/webapi/product/GetPDBIOS?website=global&model=GX650PZ&cpu=
<img width="1420" alt="Screenshot 2024-02-07 at 21 54 40" src="https://github.com/seerge/g-helper/assets/5920850/2807bb92-fb57-43bf-888d-3a8c938325cd">

**Reply by ghost** [reply]
They just didn't update their site yet, I guess.

**Comment by Kikimahe** [comment]
hi, 

Firmware 319 is on line.  :) 

![Capture d'écran 2024-02-08 200407](https://github.com/seerge/g-helper/assets/142881210/b5791bf1-2d78-4530-b7d4-9b51037ad049)
